interface/src/app/greenelab/utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * This module includes a few utility components that will be shared in other
+ * modules to avoid duplicate codes.
+ */
+
+angular.module('greenelab.utils', [])
+.constant('ApiBasePath', '/api/v0/')
```

Comment:
This is actually a constant of API path. Suppose we decide to change our backend path to `/api/v1/` in the future, without using this constant in every module, we will have to change many places in the codebase. But if we use this constant in every module that calls the API, then we only need to change this constant.

(I learned this trick from an online Angular course at coursera.)